queries.sql

```diff
@@ -1,3 +1,48 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- MONDAY
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016/01/01' and '2019/31/12';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- TUESDAY
+
+BEGIN;
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD unspecified VARCHAR(100);
+ROLLBACK;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE name NOT LIKE '%mon';
+COMMIT;
+
```

Comment:
- [ ] Kindly add the transaction that represents this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Now%2C%20take%20a%20deep%20breath%20and...%20Inside%20a%20transaction%20delete%20all%20records%20in%20the%20animals%20table%2C%20then%20roll%20back%20the%20transaction.) for deleting all records then rollback.